langlib/lang.error/src/main/java/org/ballerinalang/langlib/error/ToBalString.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *   Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.ballerinalang.langlib.error;
+
+import io.ballerina.runtime.api.utils.StringUtils;
+import io.ballerina.runtime.api.values.BError;
+import io.ballerina.runtime.api.values.BString;
+
+/**
+ * Returns expression style representation of the given value as a String.
+ *
+ * @since 2.0.0
+ */
+public class ToBalString {
```

Comment:
Have we added tests for `toBalString` and `toString` for errors?

Reply:
Tests related to `toBalString` are already there. Tests related to `toString` are added.